SeedDistroGuide.md

```diff
@@ -3,7 +3,13 @@
 ## A. Calculating distribution + updating ledger
 1. Fork the repo locally and follow the instructions in the main readme to load the SourceCred instance (you will need API keys for Github and Discord)
 2. If you already forked the repo, make sure to pull any changes from the master branch (since the discord bot updates ETH addresses in the ledger programmatically)
-3. Run `yarn load` to load the latest contribution data.
+3. Due to painfully long and unreliable method of loading contribution data locally using `yarn load`, we now use the following method from within the root of the XP repo, [as advised by SC](https://discord.com/channels/453243919774253079/718263631158050896/778428725570174986):
```

Comment:
Nice, yeah, I experimented with something similar to this at some point. Nice to have it written down somewhere!